x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/edit_policy.js

```diff
@@ -238,6 +238,12 @@ class EditPolicyUi extends Component {
                     }
                     titleSize="s"
                     fullWidth
+                    description={
```

Comment:
Based on the patterns I've seen in our other forms, I think this should be `helpText` for the input itself.

Reply:
fixed